README.md

```
@@ -2026,3 +2026,32 @@ requests](https://github.com/vacanza/holidays/pulls) are always welcome. Please
 
 Code and documentation are available according to the MIT License (see
 [LICENSE](https://github.com/vacanza/holidays/blob/dev/LICENSE)).
+
+
+This script generates holiday calendar (.ics) files for different countries using the holidays library.
+## Usage
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)